include/nanobind/nb_attr.h

```diff
@@ -156,8 +157,17 @@ template <size_t Size> struct func_data_prelim {
     /// Supplementary flags
     uint32_t flags;
 
-    /// Total number of function call arguments
-    uint32_t nargs;
+    /// Total number of arguments accepted by the C++ function; nb::args
+    /// and nb::kwargs parameters are counted as one each. If the
+    /// 'has_args' flag is set, then there is one arg_data structure
+    /// for each of these.
+    uint16_t nargs;
+
+    /// Number of arguments to the C++ function that may be filled from
```

Comment:
Just to clarify: this includes ``nb::arg()`` (unnamed) and nb::arg("name")` (named) arguments?